CHANGELOG.md

```diff
@@ -4,27 +4,51 @@ Notable changes since version 42.0.0, read the complete [History of Changes](htt
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
+### Changed
+
+### Added
+
+### Fixed
+
+## [42.2.11] (2020-03-07)
+### Changed
+ - Reverted [PR 1641](https://github.com/pgjdbc/pgjdbc/pull/1252). The driver will now wait for EOF when sending cancel signals. 
+ - `DatabaseMetaData#getProcedures` returns only procedures (not functions) for PostgreSQL 11+ [PR 1723](https://github.com/pgjdbc/pgjdbc/pull/1723)
+ - Convert silent rollbacks into exception if application sends `commit` or `xa.prepare` command [PR 1729](https://github.com/pgjdbc/pgjdbc/pull/1729)
+
+### Added
+ - Convert silent rollbacks into exception if application sends `commit` or `xa.prepare` command [PR 1729](https://github.com/pgjdbc/pgjdbc/pull/1729)
```

Comment:
Why do we have this twice

Reply:
Well, it is "feature added" and "behaviour changed" at the same time.
Feel free to remove one of them (or rephrase?)

Reply:
What do you think of mentioning the new property in the added section then?
I also now wonder if we should not version properties going forward ? For instance put @Since in the javadoc for each property ?

Reply:
Indeed. I've reworded the entry in `added` section.

In the future, we could add `@API` annotations via https://apiguardian-team.github.io/apiguardian/docs/current/api/org/apiguardian/api/package-summary.html